spec/controllers/api/emoji_controller_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Api::EmojiController, type: :controller do
+  # :FIXME APIのテストなのでrequest specが適当らしい。誰か書きなおしてください
+  describe 'Get #index', login_as: :current_user do
+    let(:current_user) { create(:user) }
+    subject { get :index, { query: emoji_name, format: :json } }
+    context '絞込した場合' do
+      let(:emoji_name) { "octocat" }
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)